code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -660,15 +671,32 @@ GLOBAL_VAR(medibot_unique_id_gen)
 						speak(message,radio_channel)
 						playsound(src, messagevoice[message], 50)
 						COOLDOWN_START(src, declare_cooldown, 10 SECONDS)
-			if(MEDIBOT_TREAT_SUCK)
+					C.visible_message("<span class='danger'>[src] injects [patient] with its syringe!</span>", \
+					"<span class='userdanger'>[src] injects you with its syringe!</span>")
+			if(MEDBOT_TREAT_BANDAGE)
+				var/mob/living/carbon/human/H = C
+				if(!H.is_bleeding())
+					to_chat(src, "<span class='warning'>[H] isn't bleeding!</span>")
+					update_icon()
+					soft_reset()
+					return
+				if(H.is_bandaged()) //so you can't stack bleed suppression
+					to_chat(src, "<span class='warning'>[H]'s bleeding is already bandaged!</span>")
+					update_icon()
+					soft_reset()
+					return
```

Comment:
People can still be bleeding while bandaged but it doesn't stack unlike the previous sytem.
```suggestion
				if(!H.is_bleeding())
					to_chat(src, "<span class='warning'>[H] isn't bleeding!</span>")
					update_icon()
					soft_reset()
					return
```